Mage/src/main/java/mage/game/command/Dungeon.java

```diff
@@ -204,7 +202,7 @@ public MageObject getCopyFrom() {
 
     @Override
     public List<CardType> getCardType(Game game) {
-        return emptyList;
+        return Collections.emptyList();
```

Comment:
Miss `DUNGEON` card type from all dungeons? It was `Collections.unmodifiableList(Arrays.asList(CardType.DUNGEON))`